files/default/neutron-ha-tool.py

```diff
@@ -322,7 +329,7 @@ def l3_agent_rebalance(qclient, noop=False, wait_for_ports=False):
             if migrate_router_safely(qclient, noop, router_id,
                                      l3_agent_dict[hgh_agent_id],
                                      l3_agent_dict[low_agent_id],
-                                     wait_for_ports):
+                                     wait_for_ports, False):
```

Comment:
I would put `delete_namespaces=False` in the signature and do not extend any new call to migrate_router_safely

In case that this value needs to be True, I would specify the parameter name in the call (`delete_namespace=True`)